app/views/landing/index.slim

```diff
@@ -0,0 +1,31 @@
+doctype html
+html
+  = render 'header'
+  body
+    div id="bar-header"
+    div id="principal" class="flex-list"
+      img id="phone" class="item medium-hide" src=asset_path("phone.png")
+      div id="bar-separator" class="item medium-hide"
+      img id="logo" class="item" src=asset_path("tcg_logo.png")
+
+    div class="decorator"
+
+    div id="features" class="flex-list"
```

Comment:
There are tags used to describe lists, for example https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl